docs/user/installation.rst

```diff
@@ -187,14 +187,7 @@ In order to work with your current infrastructure, you need to specify some envi
                    --name nginx \
                    certat/intelmq-nginx:latest
 
-   sudo docker run -e INTELMQ_IS_DOCKER="true" \
-                   -e INTELMQ_SOURCE_PIPELINE_BROKER: "redis" \
-                   -e INTELMQ_PIPELINE_BROKER: "redis" \
-                   -e INTELMQ_DESTIONATION_PIPELINE_BROKER: "redis" \
-                   -e INTELMQ_PIPELINE_HOST: redis \
-                   -e INTELMQ_SOURCE_PIPELINE_HOST: redis \
-                   -e INTELMQ_DESTINATION_PIPELINE_HOST: redis \
-                   -e INTELMQ_REDIS_CACHE_HOST: redis \
+   sudo docker run -e INTELMQ_PIPELINE_HOST: redis \
```

Comment:
This environment variable is only used in the tests, but not in the normal pipeline/bot code. Should be `INTELMQ_SOURCE_PIPELINE_HOST` and `INTELMQ_DESTINATION_PIPELINE` host IMO.

Reply:
To clarify what I meant: Setting `INTELMQ_PIPELINE_HOST` does not set the pipeline host to `redis` for the bots at runtime and is thus ineffective.